### PR TITLE
Feature/ant recursive download

### PR DESCRIPTION
**Pull request by andrea-rockt**
Implemented an ant task able to get resources from a remote server recursively.

### CODE REVIEW

**Comment by andrea-rockt** [comment]
The broken test is not relative to contributed code.

File: src/main/java/com/github/sardine/ant/command/RecursiveGet.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+ * A nice ant wrapper around sardine.list() and sardine.get().
+ * 
+ * @author andreafonti
+ */
+public class RecursiveGet extends Command {
+
+	/**
+	 * Webdav server url
+	 */
+	String fServerUrl;
```
Comment (lookfirst) [inline src/main/java/com/github/sardine/ant/command/RecursiveGet.java]: No f prefix please. =)

Comment (andrea-rockt) [inline src/main/java/com/github/sardine/ant/command/RecursiveGet.java]: I'm sorry, i began implementing the change by looking at other code related to ant tasks in order to code in the same style, noticed your comments about prefixes in the other pull requests only after i opened mine.

I will fix the pr in monday.

Thanks for your time

**Comment by andrea-rockt** [comment]
Fixed code style issues

**Comment by lookfirst** [comment]
:+1: THANK YOU!

**Pull request merged**